### PR TITLE
[MRG+1] Doc: update wording for COOKIES_ENABLED

### PR DESCRIPTION
**Pull request by stav**
I just wanted to make the description flow a little better and also add the `reqmeta` for better linking.

### CODE REVIEW

File: docs/topics/downloader-middleware.rst [hunk 1]
```diff
@@ -237,16 +237,14 @@ Default: ``True``
...
-:class:`~scrapy.http.Response` will **not** be merged with the existing 
-cookies.
-
-For more detailed information see the ``cookies`` parameter in 
-:class:`~scrapy.http.Request`
+Notice that despite the value of :setting:`COOKIES_ENABLED` setting if
+``Request.``:reqmeta:`meta['dont_merge_cookies'] <dont_merge_cookies>`
+evaluates to ``True`` the request cookies will **not** be sent to the
+web server and received cookies in :class:`~scrapy.http.Response` will
+**not** be merged with the existing cookies.
```
Comment (kmike) [inline docs/topics/downloader-middleware.rst]: 👍

**Comment by kmike** [comment]
Looks good to me, thanks @stav! 
I haven't checked how updated `:reqmeta:` looks like in the docs, but I assume you've checked it and it looks fine.

**Comment by stav** [comment]
Yep, rendered everything and it all looks good.

**Comment by lopuhin** [comment]
Thanks @stav !

**Pull request merged**